alibuild_helpers/build.py

```diff
@@ -20,6 +20,7 @@
 from alibuild_helpers.log import logger_handler, LogFormatter, ProgressPrint, riemannStream
 from datetime import datetime
 from glob import glob
+from multiprocessing.pool import ThreadPool
```

Comment:
Please add `multiprocessing` to the `setup.py` as well. While I agree this should always be there on any recent python distribution, it's better safe than sorry.